enhancements/20230504-freeze-fs-during-snapshot-creation.md

```diff
@@ -0,0 +1,69 @@
+# Freeze FS during snapshot creation
+
+## Summary
+
+HEP introduces proposed changes needed to harvester to meet the requirements of performing a fs freeze on guests when a backup is performed.
+
+Currently when a user initiates a VM backup, no guest level action is performed to ensure the filesystems are sync or io is paused.
+
+If a backup is triggered during a high IO event, the backup can be inconsistent.
```

Comment:
```suggestion
Currently, when a user initiates a VM backup, no guest-level action is taken to ensure the file systems are synced, or I/O is paused. This can lead to an inconsistent backup if the backup is triggered during a high I/O event.
```